### PR TITLE
fix: move resize event to ContainerManager

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

Move resize event to ContainerManager to develop InfiniteGrid v4.

### CODE REVIEW

File: src/ContainerManager.ts [hunk 1]
```diff
@@ -77,5 +90,33 @@ export class ContainerManager {
     if (style.position === "static") {
       container.style.position = "relative";
     }
+    if (this.options.autoResize) {
```
Comment (mixed) [inline src/ContainerManager.ts:93]: @WoodNeck 
I think this feature is also needed in Flicking, how is it implemented?

@daybrush 
Don't you need this functionality for infiniteGrid as well? For example, in the case of relative infinite, which will be developed soon, I don't think grid is used. How is this functionality implemented in infiniteGrid, and is there any possibility of interference or problems with each other?

**Review by mixed — COMMENTED** [review]
LGTM
Can't this function make a TestCase?

File: src/ContainerManager.ts (hunk 1 above)
Comment (daybrush) [reply src/ContainerManager.ts:93]: @mixed 

I plan to disable Grid's autoResize and use InfiniteGrid's autoResize.

So I moved it to ContainerManager.

Comment (WoodNeck) [reply src/ContainerManager.ts:93]: @mixed If you're talking about `autoResize`, It's already in Flicking

**Review by mixed — APPROVED** [review]

**Review by WoodNeck — APPROVED** [review]
LGTM

**Pull request merged**